resources/tidb-pdf-generation-tutorial.md

```diff
@@ -1,60 +1,71 @@
+
 
 ### Preparation 1: Install and configure the Docker environment
 
 > Estimated time: 30 minutes.
 
-The following uses macOS or Windows as an example to install Docker Desktop.
+The following steps take macOS or Windows as an example for Docker Desktop installation.
+
+
+1. Install [Docker Desktop](https://docs.docker.com/get-docker/).
 
-1. Install [Docker Desktop](https://docs.docker.com/get-docker/)。
 
 2. Run the `docker --version` command in macOS Terminal or Windows PowerShell.
 
     If you see the Docker version information, the installation is successful.
 
 3. Configure Docker resources.
 
-    1. Open the Docker application and click the gear icon in the upper-right corner.
+    1. Launch the Docker application and click the gear icon in the upper-right corner.
+
     2. Click **Resources** and set **Memory** to `8.00 GB`.
 
-4. Run the following command in macOS Terminal or Windows PowerShell to pull the Docker image used for TiDB PDF document construction:
+4. Run the following command in macOS Terminal or Windows PowerShell to pull the Docker image used for building TiDB PDF documentation:
+
 
     ```bash
     docker pull andelf/doc-build:0.1.9
     ```
 
-### Preparation 2: Clone the TiDB Documentation repository to local
+### Preparation 2: Clone the TiDB documentation repository to your local disk
+
 
 > Estimated time: 10 minutes.
 
-TiDB Documentation in Chinese repository: <https://github.com/pingcap/docs-cn>, TiDB Documentation in English repository: <https://github.com/pingcap/docs>.
+TiDB English documentation repository: <https://github.com/pingcap/docs>; TiDB Chinese documentation repository: <https://github.com/pingcap/docs-cn>
+
 
-Follow the steps to clone the TiDB Documentation in English as an example:
+The following steps take TiDB English documentation as an example to show how to clone the repository:
+
+1. Go to the TiDB English documentation repository: <https://github.com/pingcap/docs>.
 
-1. Open TiDB Documentation repository: <https://github.com/pingcap/docs>。
 
 2. Click [**Fork**](https://github.com/pingcap/docs/fork) in the upper-right corner, and wait for the Fork to complete.
 
-3. Use any of the following methods to clone the TiDB Documentation repository locally.
+3. Use either of the following methods to clone the TiDB documentation repository locally.
 
     - Method 1: Use GitHub Desktop client.
 
         1. Install and launch [GitHub Desktop](https://desktop.github.com/).
-        2. In the GitHub Desktop, click **File** > **Clone Repository**.
-        3. Click the **GitHub.com** tab and select **Your Repositories**, then select the repository you forked, click **Clone** in the lower-right corner.
+        2. In GitHub Desktop, click **File** > **Clone Repository**.
+
+        3. Click the **GitHub.com** tab, select the repository you forked in **Your Repositories**, and then click **Clone** in the lower-right corner.
+
+    - Method 2: Use the following `Git` commands.

```

Comment:
```suggestion
    - Method 2: Use the following Git commands.

```

I think it should be "Git" or "`git`".

Reply:
@dveeden thank u, I modify it to "`git`"